crates/ln-dlc-node/src/tests/dlc/dlc_setup_with_reconnects.rs

```diff
@@ -76,56 +69,47 @@ async fn reconnecting_during_dlc_channel_setup() {
     app.reconnect(coordinator.info).await.unwrap();
 
     // Instruct coordinator to re-send the accept message
-    process_pending_dlc_actions(
+    sub_channel_manager_periodic_check(
```

Comment:
> An assertion that a message is pending would be nice here. Otherwise we are just implying that this will resend the accept message

Covered with my response above.

> Also shouldn't it be the confirm message? The accept message is sent by the app.

We never adapted these tests, so we are not fully mimicking what we do in production. That is, the app proposes the channel open and close in most of these tests (if not all).